components/automate-cli/cmd/chef-automate/ha_cert_show.go

```diff
@@ -114,26 +124,26 @@ func (c *certShowImpl) certShow(cmd *cobra.Command, args []string) error {
 	if deployerType == EXISTING_INFRA_MODE {
 		config, err := c.nodeUtils.getInfraConfig(&c.sshUtil)
 		if err != nil {
-			return err
+			return nil, err
 		}
 		certInfo, err = c.getCerts(config)
 		if err != nil {
-			return err
+			return nil, err
 		}
 	} else if deployerType == AWS_MODE {
 		config, err := c.nodeUtils.getAWSConfig(&c.sshUtil)
 		if err != nil {
-			return err
+			return nil, err
 		}
 		certInfo, err = c.getCerts(config)
 		if err != nil {
-			return err
+			return nil, err
 		}
 	} else {
-		return status.New(status.ConfigError, "Invalid deployer type. Either architecture.existing_infra or architecture.aws must be set in config.toml")
+		return nil, status.New(status.ConfigError, "Invalid deployer type. Either architecture.existing_infra or architecture.aws must be set in config.toml")
 	}
 
-	return c.validateAndPrintCertificates(remoteService, certInfo)
+	return certInfo, nil
```

Comment:
why we have removed `validateAndPrintCertificates`

Reply:
its not removed its just shifted.